__azurite_db_table__.json

```diff
@@ -0,0 +1,114 @@
+{
+  "filename": "/Users/deepanshumarwah/projects/quantum-sec/Azure-Sentinel/__azurite_db_table__.json",
```

Comment:
Ouch, didn't notice this before

Reply:
oh shoot

Reply:
removed this and also there was a space issue in app kind.